docs/ldml/tr35-messageFormat.md

```diff
@@ -3351,7 +3353,7 @@ The following options and their values are required to be available on the funct
    -  `plural` (default)
    -  `exact`
 - `currency`
-   - well-formed [Unicode Currency Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCurrencyIdentifier)
+   - well-formed [Unicode Currency Identifier](./tr35.md#UnicodeCurrencyIdentifier)
```

Comment:
Why does this tr35.md have a ./ at the beginning when the rest don't?

Reply:
it should not have ./